easybuild/tools/config.py

```diff
@@ -66,6 +66,8 @@
 DEFAULT_PREFIX = os.path.join(os.path.expanduser('~'), ".local", "easybuild")
 DEFAULT_REPOSITORY = 'FileRepository'
 
+PREFERRED_JOB_BACKENDS = ['Pbs', 'GC3Pie']
```

Comment:
rename the constant (or add comment) to reflect that the order is relevant (i.e. not just a list of possibilities)